frontend/src/container/ListOfDashboard/DashboardsList.tsx

```diff
@@ -704,6 +705,13 @@ function DashboardsList(): JSX.Element {
 							intercomMessageDisabled
 						/>
 					</Flex>
+					{isCloudUser() && (
```

Comment:
Should we enable this to EE users too ?

Reply:
how can we enable, can you please the reference logic or code ?

Reply:
No we don't have analytics for non cloud users so no need to enable this for EE